lms/services/product.py

```diff
@@ -0,0 +1,24 @@
+from lms.models import Product
+
+
+def _get_product(request):
+    if request.content_type == "application/json":
```

Comment:
In the API we don't need to guess, we could tell ourselves (ie send the data to the frontend and get it back when it calls us).

This will help having generic version of some api endpoints, /groups/ vs /canvas/groups /blackboard/groups